app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt

```diff
@@ -678,10 +697,18 @@ fun CommentFooterLine(
             onEditCommentClick = onEditCommentClick,
             onDeleteCommentClick = onDeleteCommentClick,
             onReportClick = onReportClick,
+            onRemoveClick = onRemoveClick,
             onBlockCreatorClick = onBlockCreatorClick,
             onCommentLinkClick = onCommentLinkClick,
             onPersonClick = onPersonClick,
             isCreator = account.id == commentView.creator.id,
+            canMod =
```

Comment:
I'm not sure at what level it makes sense to run this function. Its really only used in the comment dropdown.

Reply:
This is alright, just make sure you surround it with a `remember`